src/resolvers/projectVerificationFormResolver.ts

```diff
@@ -183,19 +184,10 @@ export class ProjectVerificationFormResolver {
       await projectVerificationForm.save();
 
       const callbackUrl = `https://${dappUrl}/verification/${project.slug}/${token}`;
-      const emailConfirmationData = {
-        email,
-        callbackUrl,
-      };
-
       await sendMailConfirmationEmail(email, project, token);
-
-      SegmentAnalyticsSingleton.getInstance().track(
-        NOTIFICATIONS_EVENT_NAMES.SEND_EMAIL_CONFIRMATION,
-        `givethId-${userId}`,
-        emailConfirmationData,
-        null,
-      );
+      await getNotificationAdapter().projectSendEmailConfirmation({
```

Comment:
I added this before but I think we can remove it. Segment doesn't send any email as we send it with:
```
      await sendMailConfirmationEmail(email, project, token);
```

Also tracking this data is not required, and thanks for removing the callbackUrl from the analytics, that info shouldn't be exposed.

I am all for deleting code as we can speed up requests.

Reply:
Nice, thanks for mentioning that, I remove it from both notificaiton-center and impact-graph